src/types/filterTypes.ts

```diff
@@ -0,0 +1,30 @@
+export type locations = string[]
+
+export type selectedFilters = {
+  location: string[]
+  format: string[]
+  status: string[]
+}
+export interface ItemAggregationOption {
+  value: string
+  count: number
+  label: string
+}
+
+export interface ItemAggregation {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  "@type": string
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  "@id": string
+  id: string
```

Comment:
Out of curiosity, what's the purpose of the second id field?

Reply:
maybe @nonword could weigh in here. The @ properties i believe refer to linked json values that are attached to a fixed URI somewhere... and that is the extent of what i can say about json ld. i assume the un-prefixed id is there for convenience on the front end.

Reply:
Yes the `@id` is a json-ld entity id, so it's theoretically namespaced to make it unique in the graph (e.g. 'resources:1001'). Entities in JSON-LD have to define an `@id`. The `id` property is the same thing but not namespaced (e.g. '1001'), so can be used without needing to drop the namespace